cli/flags.go

```diff
@@ -178,4 +183,19 @@ var (
 		Value:    5,
 		Category: RelayCategory,
 	}
+
+	// metrics
+	metricsFlag = &cli.BoolFlag{
+		Name:     "metrics",
+		Sources:  cli.EnvVars("METRICS_ENABLED"),
+		Usage:    "enables a metrics server",
+		Category: Metrics,
+	}
+	metricsAddrFlag = &cli.StringFlag{
+		Name:     "metrics-addr",
+		Sources:  cli.EnvVars("METRICS_ADDR"),
+		Value:    "localhost:18551",
+		Usage:    "listening address for the metrics server",
+		Category: Metrics,
+	}
```

Comment:
minor nit: it would be nice to separate it out to metrics-addr and metrics-port into separate flags.